tests/st/utils/data.py

```diff
@@ -261,14 +261,6 @@
             'name': 'endpoint1',
             'uid': 'test-uid-change',
         }],
-        'initializers': {
```

Comment:
Can you provide some reasoning why this removal is needed? I think removing it was for fixing a couple tests but I'd like to know why that removal is ok.

Reply:
That field was deprecated in 1.13 and removed in 1.16.0